x/feemarket/keeper/migrations.go

```diff
@@ -1,13 +1,10 @@
 package keeper
 
 import (
-	"math/big"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
-)
 
-// KeyPrefixBaseFeeV1 is the base fee key prefix used in version 1
-var KeyPrefixBaseFeeV1 = []byte{2}
+	v010 "github.com/tharsis/ethermint/x/feemarket/migrations/v010"
```

Comment:
what's up with the versioning for this pacakge?

Reply:
why do we need a JSON migration here? when is this pattern used?

Reply:
there's a `"migrate"` command on the SDK to migrate the genesis.json file but it's not widely used. I asked about it in https://github.com/cosmos/cosmos-sdk/issues/11504